introduction.md

```diff
@@ -0,0 +1,144 @@
+
+The single-tiered multi-signature approach common today is subject to a trade-off between security (large number of signatures requried) and usability (low number of parties required). By comparison, Revault achieves practical usability through safe delegation to managers who handle the spending process without sacrificing on the security of a large multi-signature.
+
+In addition, Revault comes with an (optional) deterrent feature for emergency scenarios. Before deploying Revault, stakeholders set-up an _emergency vault_. At any time stakeholders can trigger their panic button and unilaterally transfer all bitcoin in custody to the emergency vault. For example if 2 of the 3 stakeholders are abducted or physically threatened, the remaining stakeholder can save funds from theft. This feature drastically reduces the likelihood that a coordinated attack on the stakeholders will be profitable and thus acts as a credible deterrent from physical coercion. 
+
+## Process and Revault Transaction Types
+
+![Revault Transaction Graph. ](Revault-Tx-Diagram.png)
+
+Funds enter custody through Deposit transactions (TXs) as unspent-transaction-outputs (UTXOs) locked to the `deposit output descriptor`, an N-of-N multi-signature among stakeholders. After receiving a deposit, the stakeholders exchange signatures for a set of 4 transactions which are _not broadcast_. These "pre-signed" transactions are used by stakeholders to control the flow of funds and enable the delegation and emergency deterrent features of Revault. As seen in the diagram, they include; Emergency, Unvault, Cancel and Unvault Emergency TXs. 
+
+The Emergency TX is the first to be signed. It would spend a deposit and pay to the `emergency output descriptor` which is a bitcoin script that is harder to unlock than `deposit output descriptor`. Keeping the `emergency output descriptor` private among stakeholders strengthens the emergency deterrent feature as attackers don't know how they would compromise the emergency vault.
+
+Then, stakeholders exchange signatures for the Cancel and Unvault Emergency TXs:
+
+- The Unvault Emergency TX would spend the Unvault TX to pay to the `emergency output descriptor`.
+- The Cancel TX would spend the Unvault TX to pay back to the `deposit output descriptor`.
+
+This way, funds are protected _before_ delegation to fund managers. Finally, the Unvault TX is signed. It spends the Deposit TX and pays to either:
+
+- the managers' K-of-M multi-signature + J cosigning servers + time-lock of X blocks, or 
```

Comment:
I think a setup with a number of cosigs different than the number of stakeholders (or say one operated by an independent organization instead of the stakeholders themselves) makes sense, but it's not actually specified nor part of Revault. Like watchtowers with only Emergency or Cancel. Therefore i don't think we should mention having a different number of cosigs here.

Reply:
Yeah. I'm adding "J = 0 if deploying without co-signing servers, J = N if deploying with co-signing servers"